src/lib/authentication/msalAuthProvider.ts

```diff
@@ -0,0 +1,207 @@
+import * as msal from "@azure/msal-browser";
+import { AccountInfo } from "@azure/msal-browser";
+import { AuthProvider } from "./authProviderInterface";
+import { LoginStatus, MSALConfig, UserAccountInfo } from "./authTypes";
+import * as R from "ramda";
+
+export const scopes = [""];
+export const staticMsalConfig: msal.Configuration = {
```

Comment:
this should be in Application code, not embedeed in the "Library", in the Provider  module.

please move this to "main"

if instead serve as a default, do not export it add do coalesce with provided overrides in the ctor.

Reply:
this are not expected to be part of the library.
please move to the App area nonetheless.